### DISCUSSION TITLE
Python versiyasi

### DISCUSSION THREAD

**Discussion by ulugbekarch** [ochiqai/python#4] [Q&A] [answered]
pythonning qaysi versiyasidan foydalanamiz?

**Comment by ochiqai** [comment, marked as answer]
**Qisqacha**: python 3.6+

Ushbu kursda biz python 3 versiyasidan foydalanamiz. python 3 ning ko'p versiyasi bor (katta va kichik). Bu kursda python 3.6 versiyasi yoki undan yuqorisi tavsiya qilinadi.

**Reply by ulugbekarch** [reply]
raxmat
